### DISCUSSION TITLE
Hide the "--8<-- [start:name]" at source when using snippet sections

### DISCUSSION THREAD

**Discussion by kristiangronberg** [facelessuser/pymdown-extensions#2025] [Q&A]
Is there a way to hide the "--8<-- [start:name]" and corresponding "end" tags, specifying the [named snippet sections](https://facelessuser.github.io/pymdown-extensions/extensions/snippets/#snippet-sections) at the source file.

**Comment by facelessuser** [comment]
What version of this library are you using, and can you give an example illustrating exactly what you are looking for? I believe the latest version does what you want, but I'm unclear from the limited info provided.
